### PR TITLE
Feat: Added Upvote  feature in posts

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR i have added the upvote feature and fixed some ui 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@app/pages/Help/HelpPage.ViewModel.ts [hunk 1]
```diff
@@ -218,7 +218,15 @@ class HelpPageViewModel {
...
     const response = await this.cannyService.listPosts(boardID, sort, search,status );
-    return response;
+    let voteList = await this.listVote();
+  let  result = response.data.posts.map( (post) => {
+
+      const isLiked = voteList.data.votes.some(
+        (vote) => vote.post.id === post.id,
+      );
+      return { ...post, isPostLiked: isLiked };
+    });
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Help/HelpPage.ViewModel.ts:228]: comment required

File: src/packages/@app/pages/Help/HelpPage.ViewModel.ts [hunk 2]
```diff
@@ -438,45 +446,16 @@ public deleteVote= async (postID:string) =>{
...
-    }
-
     const UserId = userResponse?.data?.id;  // Use the retrieved or newly created user's ID
 
     if (UserId) {
-      const result = await this.cannyService.listVotes(postID);
+      console.log("This is user ID", UserId)
+      const result = await this.cannyService.listVotes(UserId);
       return result;
     }
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Help/HelpPage.ViewModel.ts:455]: logs not required

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**